modmail/extensions/utils/error_handler.py

```diff
@@ -0,0 +1,195 @@
+import logging
+import re
+import typing
+
+import discord
+import discord.errors
+from discord.ext import commands
+
+from modmail.bot import ModmailBot
+from modmail.log import ModmailLogger
+from modmail.utils.cogs import BotModes, ExtMetadata, ModmailCog
+from modmail.utils.extensions import BOT_MODE
+
+
+logger: ModmailLogger = logging.getLogger(__name__)
+
+EXT_METADATA = ExtMetadata()
+
+ERROR_COLOUR = discord.Colour.red()
```

Comment:
Does this need to be a constant? It's just used once.

Reply:
Yes, in a future pr I'll be moving it to the configuration system.